static/css/components/fulltext-suggestion-snippet.less

```diff
@@ -0,0 +1,44 @@
+.fsi-snippet {
+  display: inline;
+  width: 60%;
+  padding: 5px;
+  margin: 0;
+  font-size: small;
+  @media (min-width: @width-breakpoint-mobile-m) {
+    width: 90%;
+  }
+  @media (min-width: @width-breakpoint-tablet) {
+    width: auto;
+    font-size: medium;
+  }
+
+  &__main {
+    padding: 5px;
+    margin: 0 5px;
+    line-height: 1.6em;
+    a:link,
+    a:hover,
+    a:visited {
+      text-decoration: none;
+    }
+    a:link {
+      font-family: @georgia_serif-1;
+      color: @black;
+    }
+    a:hover {
+      color: @link-blue;
+    }
+  }
+
+  &__quotation-mark {
+    font-size: larger;
+    display: inline;
+  }
+
+  &__page-num {
+    color: @dark-grey !important;
+    display: inline;
+    text-decoration: none !important;
+    font-family: @lucida_sans_serif-1 !important;
```

Comment:
We want to avoid `!important` here by setting parent classes appropriately

Reply:
I believe `&__page-num a` should do the trick https://chatgpt.com/share/a13d28ab-8bf2-49bf-b22f-a8da99a01cc6